ephios/settings.py

```diff
@@ -311,5 +312,25 @@ def GET_SITE_URL():
     "DEFAULT_AUTHENTICATION_CLASSES": [
         "rest_framework.authentication.TokenAuthentication",
```

Comment:
I think we should remove the Token Auth with this PR, as we really want to transition to our own Token Model. 

* remove Token Auth
* remove Token app
* migration (?)